frontend/src/components/ResizeTable/ResizeTable.tsx

```diff
@@ -63,8 +64,10 @@ function ResizeTable({
 			...restProps,
 			components: { header: { cell: ResizableHeader } },
 			columns: mergedColumns,
+			hideOnSinglePage: true,
+			pagination: { ...pagination, hideOnSinglePage: true },
```

Comment:
also since we are always overriding `hideOnSinglePage: true` being passed from the parent do we need to send the same from the parent ?

Reply:
Thanks for pointing this out :+1: :pray: . 

No we don't need to pass it from the parent. 

Discarded the changes where I was passing it from the parent :slightly_smiling_face: .